src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area, Drawable {
+    private final String colour;
+
+    protected Figure(String colour) {
+        this.colour = colour;
+    }
+
+    protected double roundValues(double value) {
+        return (double)Math.round(value * 1000d) / 1000d;
+    }
+
```

Comment:
```suggestion
    protected double roundValue(double value) {
        return (double) Math.round(value * 1000d) / 1000d;
    }

```

Reply:
Actually, this is a violation of the Single Responsibility Principle. Figure shouldn't be responsible for rounding values. Let's use existing methods or create a separate service class for this purpose